dm/worker/worker.go

```diff
@@ -741,6 +741,9 @@ func (w *Worker) restoreSubTask() error {
 			return terror.Annotatef(err, "decode subtask config %s error in restoreSubTask", task.Task)
 		}
 
+		// when restarting DM-worker, some config items may need to be updated for the previous saved task config.
+		w.copyConfigFromWorker(taskCfg)
```

Comment:
do we need to remove the check since we do `copyConfigFromWorker` here
 https://github.com/pingcap/dm/blob/487dc85ef0144d617089d90fc4379b9da76a4088/syncer/heartbeat.go#L106